buildSrc/src/main/kotlin/org/jetbrains/exposed/gradle/conventions/dokka.gradle.kts

```diff
@@ -0,0 +1,5 @@
+package org.jetbrains.exposed.gradle.conventions
+
+plugins {
+    id("org.jetbrains.dokka")
+}
```

Comment:
Even though the rest of the project uses `allprojects {}` and `subprojects {}`, I decided to go with a convention plugin so that it doesn't have to be refactored later on.

Can quickly change it to the `allprojects {}` approach though, if you ask.